src/common-utils/main.sh

```diff
@@ -393,7 +393,7 @@ fi
 possible_rc_files=( ".bashrc" ".profile" ".zshrc" )
 for rc_file in "${possible_rc_files[@]}"; do
     if [ -f "/etc/skel/${rc_file}" ]; then
-        if [ ! -e "${user_rc_path}/${rc_file}" ] || [ -s "${user_rc_path}/${rc_file}" ]; then
```

Comment:
Can we bump the patch version here? Thanks!

Reply:
Sure, I forgot about that! Feel free to squash the additional commit when merging the PR.